schemas/gtfs_realtime_source_schema.json

```diff
@@ -28,6 +28,10 @@
       "type": "string",
       "description": "An optional description of the data source, e.g to specify if the data source is an aggregate of multiple providers, or which network is represented by the source."
     },
+    "feed_contact_email": {
```

Comment:
@emmambd, should we allow multiple emails?

Reply:
@davidgamez Single email to reduce confusion = the goal is that consumers know the 1 point of contact they should go to for issues with the feed.

Reply:
Fair enough.